app/Http/Controllers/CampaignsController.php

```diff
@@ -107,13 +106,15 @@ public function showCampaign($id)
 
         return view('pages.campaign_single')
             ->with('state', [
-                'signups' => $signups,
+                'signups' => $signups->toArray()['data'],
```

Comment:
Do you need to grab from `['data']` here? I think that comes from the transformer, not model's `toArray`.